### PR TITLE
Add support for VSCode Insiders build

### PR DESCRIPTION
**Pull request by LyzardKing**
Merge #320
@didrocks The large test is working.
The medium test fails, it can't find the process to stop.
In the container the process seems to be working properly as far as I can tell.

### CODE REVIEW

File: tests/medium/__init__.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def check_and_kill_process(self, process_grep, wait_before=0, send_sigkill=False
                                                                                       "check_and_kill_process"),
                                                                          send_sigkill,
                                                                          " ".join(process_grep))))[0]:
-            raise BaseException("The process we try to find and kill can't be found".format(process_grep))
+            raise BaseException("The process we try to find and kill can't be found: {}".format(process_grep))
```
Comment (LyzardKing) [inline tests/medium/__init__.py]: Was this omitted for a reason?

Comment (didrocks) [inline tests/medium/__init__.py]: No, it wasn't :)

**Comment by LyzardKing** [comment]
in the install folder bin/code calls code.
So we'd need to close code instead of bin/code.
At least, this is what works with the large tests.
I can't get the medium tests to work now...

**Comment by didrocks** [comment]
As discussed by emails, the medium tests actually works! Merging this thus :) Thanks!

**Pull request merged**